### ISSUE TITLE
Animation Blueprints Nodes (not working on latest) [UE5]

### ISSUE THREAD

**Issue by Tectors** [FabianFG/CUE4Parse#48]
Since the release of Unreal Engine 5, all animation blueprints are random when it comes to showing the nodes, sometimes the animation blueprint shows like 2 nodes out of 500 and sometimes doesn't show any nodes.

Usually the nodes show if the animation blueprint wasn't opened by the Unreal Editor (from the creator's perspective), and would only show the nodes that weren't modified from UE4 to UE5.

UE5 changed something related to animation blueprint nodes, and we can't see anything in them. I dearly ask for one of the contributors of CUE4Parse to fix this as this would be such a great help for me and others.

Here below is a preview of the changes that happened between UE4 to UE5 related to the JSON output of CUE4Parse.

-------------

_This is exported using FModel, but that won't change anything related to the JSON as it uses CUE4Parse to parse assets._

| Unreal | Fortnite Version |
| ----------- | ----------- |
| UE4_25 | Fortnite Version 12.41 |
| UE5_1 | Fortnite Version 20.30 |

| Asset |
| ----------- |
| Characters/Player/Male/Medium/Bodies/M_Med_Soldier_04/Meshes/SK_M_Med_Soldier_04_Skeleton_AnimBP |

[Comparison_AnimBP_UE4_to_UE5.zip](https://github.com/FabianFG/CUE4Parse/files/8839035/Comparison_AnimBP_UE4_to_UE5.zip)

---------

### Unreal Engine 4.25.0

The animation blueprint provides every single bit of animation nodes and leaves out nothing.

--------
### Unreal Engine 5.1.0

The animation blueprint leaves out almost to no nodes, only a copy pose from mesh node (which is utterly useless).

-----------

### Conclusion

In conclusion, the animation blueprints change between UE4 to UE5 removing all animation nodes and anything valuable to use, causing sometimes a empty object or one node in the output JSON.

**Comment by iAmAsval** [comment]
blame mappings

**Comment by Tectors** [comment]
> blame mappings

fix em

**Comment by iAmAsval** [comment]
> 

diy

**Comment by Tectors** [comment]
> diy

hmmmmmm i'll see what i can do myself

**Comment by Tectors** [comment]
> > fix em
> 
> diy

could u add me on discord i wanna ask a question 


Tector#7383

**Comment by Tectors** [comment]
i'm just gonna leave it up to you guys to fix it since i know way less about this stuff
👍🏽

**Comment by Tectors** [comment]
i just wanted to quickly say, it is very dumb you guys haven't updated the mappings (it's a whole new unreal engine version are you serious?)
